app/assets/javascripts/homepage/conversation.coffee

```diff
@@ -25,6 +25,7 @@ conversation.controller("ConversationController", ["$scope", "$stateParams", "AP
     conversationPageState = READ_POSTS
     previousPostContainerOffset = 0
     # Scope methods and models
+    $scope.username = AppState.user.username
```

Comment:
why is this bound to the scope? As far as I can tell `conversation.html` doesn't use this anywhere.